_specs/ecip-1077.md

```diff
@@ -1,7 +1,7 @@
 ---
 ecip: 1077
 title: The Meta Fork ECIP: Establishing _Fork_ type ECIPs and their specifications and processes
-author: Mr. Meows D. Bits
+author: Mr. Meows D. Bits (@meowsbits)
```

Comment:
```suggestion
author: Mr. Meows D. Bits
```

Reply:
Just following the ECIP-1000

"The Author header lists the names and email addresses/GitHub handles of all the authors/owners of the ECIP. The format of the Author header value must be:

Random J. User <123@dom.ain>
Random J. User (@random.j.user)
If there are multiple authors, each should be on a separate line following RFC 2822 continuation line conventions."

Reply:
Just following the ECIP-1000

The Author header lists the names and email addresses/GitHub handles of all the authors/owners of the ECIP. The format of the Author header value must be:

Random J. User <123@dom.ain>
Random J. User (@random.j.user)
If there are multiple authors, each should be on a separate line following RFC 2822 continuation line conventions.

Reply:
I have a different interpretation: It does not mean than an author MUST register an email (OR a Github handle, now); it means that IF an author registers one of those values, they should be formatted as described.